exonum/src/runtime/rust/tests.rs

```diff
@@ -198,7 +198,13 @@ impl<T: Runtime> Runtime for Inspected<T> {
         context: ExecutionContext<'_>,
         instance_id: u32,
     ) -> Result<(), ExecutionError> {
-        let height = CoreSchema::new(&*context.fork).height();
+        let schema = CoreSchema::new(&*context.fork);
+        // Skip genesis block execution.
```

Comment:
Nit: This can be more accurately worded as "Skip adding events during genesis block execution". The `inner` runtime is notified about the call in any case.